modulemd/tests/test-modulemd-merger.c

```diff
@@ -20,6 +21,7 @@
 #include "modulemd-module-index.h"
 #include "private/test-utils.h"
 
+typedef int gint;
```

Comment:
You could also use `guint64` and not use this at all. This is all defined in `glib` already.